scripts/zones/Spire_of_Dem/mobs/Ingester.lua

```diff
@@ -29,23 +21,27 @@ entity.onMobInitialize = function(mob)
 end
 
 entity.onMobWeaponSkillPrepare = function(mob, target)
+    -- 20% chance to prefer Fission
+    if math.random(1, 100) <= 20 then
+        local skillList = { xi.mobSkill.FISSION }
+        skillList = xi.mix.gorger.filterFission(mob, skillList)
+
+        if #skillList > 0 then
+            return skillList[1]
+        end
+    end
```

Comment:
This is a good example of how just having the function check if fission is possible would simplify the logic, i think.